test/MLOps.NET.Tests/Deployments/SchemaGeneratorTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MLOps.NET.Docker;
+using MLOps.NET.Docker.Interfaces;
+using MLOps.NET.Docker.Settings;
+using MLOps.NET.Entities.Impl;
+using MLOps.NET.Storage.Deployments;
+using Moq;
+using System.IO;
+using System.IO.Abstractions;
+using FluentAssertions;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.Tests.Deployments
+{
+    [TestClass]
+    public class SchemaGeneratorTests
+    {
+        private SchemaGenerator sut;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            this.sut = new SchemaGenerator();
+        }
+
+        [TestMethod]
+        public void GenerateDefinition_ShouldGenerateModelOutput()
+        {
+            //Arrang
+
+            //Act
+            var modelOutput = sut.GenerateDefinition<ModelOutput>("BinaryClassificationModelOutput");
+
+            //Assert
+            modelOutput.Should().Contain("using Microsoft.ML.Data;");
+            modelOutput.Should().Contain("namespace MLOps.NET.Tests.Deployments");
+            modelOutput.Should().Contain("public class BinaryClassificationModelOutput");
+            modelOutput.Should().Contain("public bool Prediction");
```

Comment:
Nice! I'm happy to merge this for now and have a look at it more closely. One thing I think would be awesome is to split this class out until it's own file, and then assert each row (including e.g. `[ColumnName("PredictedLabel"]` etc.). This is a good start though if you just want me to merge it :)

Reply:
done